operate/client/e2e-playwright/tests/processInstanceVariables.spec.ts

```diff
@@ -47,8 +47,8 @@ test.beforeAll(async ({request}) => {
 });
 
 test.describe('Process Instance Variables', () => {
-  test('Edit variables', async ({page, processInstancePage}) => {
-    test.setTimeout(DEFAULT_TEST_TIMEOUT + 1 * 15000); // 15 seconds for each applied operation in this test
```

Comment:
I removed this in favor of `test.slow()`.

Before: 45s timeout (30s + 15s)
After: 90s timeout (30s * 3)